gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/common/dialect/AbstractDialectHelper.java

```diff
@@ -318,8 +318,8 @@ protected StringBuilder buildSearchUser(boolean wildcard, StringBuilder builder)
     }
 
     @Override
-    public String buildFindUserByDomainAndEmail(ReferenceType referenceType, String referenceId, String email, boolean strict) {
-        return new StringBuilder("SELECT * FROM users u WHERE ")
+    public String buildFindUserByDomainAndEmail(ReferenceType referenceType, String referenceId, String email, boolean strict, boolean organizationUser) {
```

Comment:
Maybe you should rename this method by buildFindUserByReferenceAndEmail.

The boolean organizationUser is really needed here ? can we not use the referenceType ?